.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: required
+dist: trusty
```

Comment:
i don't think this matters with tox

Reply:
All others projects are using it, not 100% if it is correct or not.
Why do you think we do not need it?
https://docs.travis-ci.com/user/reference/overview/#Virtualisation-Environment-vs-Operating-System

Reply:
With sudo: required and dist: trusty we will get Ubuntu 14.04 VM, that sounds like good solution for NM testing, but it will not work (IMO), NM on Ubuntu 14 would be really outdated and API will be different with missing features and bugs.

I suggest we use sudo: false dist: thrusty to get a container and do proper NM testing either in a VM or run it on Jenkins.

Reply:
Ok, so this line stays the same anyway.
Regarding the NM version, we will have to solve this somehow, but I hope we can do it on travis.